### PR TITLE
More flexible slow PR CI

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Taken from  #33170

### CODE REVIEW

File: .github/workflows/self-comment-slow-ci.yml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - name: Set up GitHub CLI
+        uses: actions/checkout@v4
+
+      - name: Reply to the comment
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```
Comment (ydshieh) [inline .github/workflows/self-comment-slow-ci.yml:46]: /pytest

**Review by ydshieh — COMMENTED** [review]
/pytest

File: .github/workflows/self-comment-slow-ci.yml (hunk 1 above)
Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml:46]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml:46]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pybon

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pybonbon

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pybonbon

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pytest

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: /pybonbon

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml]: run-slow: vit, altclip

**Comment by ArthurZucker** [comment]
🤣

**Comment by ydshieh** [comment]
i changed it to run-slow so far, pytest is not covered yet Sir!

**Comment by ArthurZucker** [comment]
Go to bed sir 🛌🏻

**Comment by ydshieh** [comment]
and comment on PR page instead of PR diff page won't work unless we merge this PR - (that's why you told me to test on another repo, which is [here](https://github.com/huggingface/transformers_ci_bot)

**Comment by ydshieh** [comment]
> Go to bed sir 🛌🏻

I would like to but my brain is full of these github action conditions 😭 

Good night ~

**Comment by ArthurZucker** [comment]
🤗

File: .github/workflows/self-comment-slow-ci.yml (hunk 1 above)
Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml:67]: run-slow: vit, altclip

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml:65]: run-slow: vit, altclip

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml:65]: run-slow: vit, altclip

Comment (ydshieh) [reply .github/workflows/self-comment-slow-ci.yml:65]: run-slow: vit, altclip

**Pull request closed without merging**